backend/src/main/java/com/redhat/cloud/notifications/auth/rbac/RbacIdentityProvider.java

```diff
@@ -108,6 +110,9 @@
                                                 })
                                                 // Otherwise, we can finish building the QuarkusSecurityIdentity and return the result
                                                 .onItem().transform(rbacRaw -> {
+                                                    if (rbacRaw.canRead("events", "events")) {
```

Comment:
```suggestion
                                                    if (rbacRaw.canRead("notifications", "events")) {
```
Can't really remember the order, but one is the "application" and other is the "resource"

Reply:
The first one is the application. The second one is an item.. maybe the parameter has got a misleading name.. probably this is the resource.

How should this be changed?

Reply:
Item should be the resource (or I confused the RBAC lingo). So my suggestion looks OK